src/routes/jira/connect/enterprise/jira-connect-enterprise-post.ts

```diff
@@ -127,7 +127,7 @@ export const JiraConnectEnterprisePost = async (
 			apiKeyHeaderName
 				? {
 					headerName: apiKeyHeaderName,
-					apiKeyGenerator: () => Promise.resolve(apiKeyValue)
+					apiKeyGenerator: () => Promise.resolve(apiKeyValue || "undefined")
```

Comment:
actually I think this must be undefined, otherwise "undefined' string will be used as an apiKeyValue (undefined means no value here)